### PR TITLE
Sides page

### PR DESCRIPTION
**Pull request by luchok22**
Issue #:
[Trello](https://trello.com/invite/b/AklPlVwg/ATTI9fdb094d2c14954c792b9e3930d9949a11CEF7C2/solid-practicum-dominos-team-2)

## PR type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### before:
dasdasСнимок

#### after:
asdasСнимок
image
image

## Does this introduce a breaking change?

- [x] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [ ] Responsive layout and functionality have been tested
  - [ ] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/component/Cart/Cart.scss [hunk 1]
```diff
@@ -1,10 +1,3 @@
-.cart-f {
```
Comment (Azim-941223) [inline src/component/Cart/Cart.scss:1]: все module.scss а этот файл почему просто scss? переимеуйте все должны быть  стандартными

File: src/component/Header/Header.jsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ function Header({ clicked, isClicked }) {
       <div className={styles.bottom}>
        <div>
       
-       <Link to="/Domino" className={styles.pizza}>
+       <Link to="/" className={styles.pizza}>
        <img  src={pizza} alt="" />
```
Comment (Azim-941223) [inline src/component/Header/Header.jsx]: alt обязательно нужно добавлять

File: src/page/PizzaPage.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import bestPrice from './../component/Array/pizzaBestPrice.json'
+import heroes from './../component/Array/pizzaHeroes.json'
+
+const PizzaPage = () => {
+    const category = [asian, bestPrice, heroes]
+
+    return (
+        <div>
+            <Header/>
+            <div className="content">
```
Comment (aiskairr) [inline src/page/PizzaPage.jsx]: Используй модульный scss

File: src/component/PizzaCart/PizzaCart.module.scss [hunk 4]
```diff
@@ -207,37 +195,39 @@
...
-        width: 100%;
-        flex-direction: column;
-
-        .cart {
-            width: 100%;
-        }
-    }
-}
\ No newline at end of file
+// @media (max-width: 990px) {
```
Comment (aiskairr) [inline src/component/PizzaCart/PizzaCart.module.scss]: удаляй закомментированный код

File: src/component/PizzaCart/PizzaCart.module.scss [hunk 5]
```diff
@@ -207,37 +195,39 @@
     }
 
     div {
-        margin-top: 10px;
-        justify-content: space-around;
+        margin: 20px 0 0 1px;
+        font-family: Open Sans, sans-serif;
```
Comment (aiskairr) [inline src/component/PizzaCart/PizzaCart.module.scss]: сделай этот стиль глобальным

File: src/component/SidesCart/SidesCart.module.scss [hunk 6]
```diff
@@ -165,10 +157,12 @@
     }
 
     div {
-        justify-content: space-around;
+        margin: 20px 0 0 10px;
+        font-family: Open Sans, sans-serif;
```
Comment (aiskairr) [inline src/component/SidesCart/SidesCart.module.scss]: этот тоже

File: src/page/Pages.module.scss [hunk 7]
```diff
@@ -0,0 +1,49 @@
+.container{
+    width: 80%;
+    margin: 0 auto;
+}
+    
+.text{
+    text-align: center;
+    font-size: 24px;
+    font-weight: 600;
+    font-family: Roboto, sans-serif;
```
Comment (aiskairr) [inline src/page/Pages.module.scss]: сделай этот стиль глобальным

File: src/page/PizzaPage.jsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <div>
+            <Header/>
+            <div className={scss.content}>
+                {category.map((arr) =>
+                    arr.map((item) =>
+                        <div className={scss.main_grid}>
+                            <p className={scss.text}>{item.text}</p>
+                            <div className={scss.cart_f}>
+                                {item.pizzas.map((i) => <Cart {...i} />)}
```
Comment (aiskairr) [inline src/page/PizzaPage.jsx]: передай уникальный ключ в компоненту Cart

File: src/page/SidesPage.jsx [hunk 9]
```diff
@@ -6,17 +6,18 @@ import chicken from './../component/Array/sidesChicken.json'
...
+            <div className={scss.content}>
                 {category.map((arr) =>
                     arr.map((item) =>
-                        <div className='MAIN'>
-                            <p className='text'>{item.text}</p>
-                            <div className='cart-f'>
+                        <div className={scss.main_grid}>
+                            <p className={scss.text}>{item.text}</p>
+                            <div className={scss.cart_f}>
                                 {item.sides.map((i) => <SidesCart {...i} />)}
```
Comment (aiskairr) [inline src/page/SidesPage.jsx]: тут тоже

**Review by aiskairr — APPROVED** [review]

**Pull request merged**